packages/datatrak-web/src/features/Survey/Components/SurveyQRCode/SurveyQRCode.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import { useLocation } from 'react-router';
+import { QRCodeList } from './QRCodeList';
+import { QRCodeModal } from './QRCodeModal';
+import { QRCodePanel } from './QRCodePanel';
```

Comment:
I think these components are really nicely split up and organised. I'm keen to avoid nesting the folder structure too deeply and I'm keen to avoid using folders named `components` if possible as I think it's not that clear to code readers. So for this one could we just move the `SurveyQRCode` folder up one level to simply be `Survey/SurveyQRCode`?